### DISCUSSION TITLE
GPG Signatures?

### DISCUSSION THREAD

**Discussion by muellert** [libgd/libgd#789] [Q&A]
Hi,

do you perchance provide some way to check for integrity?

I'm interested in signatures for the source packages.

Thanks!

**Comment by vapier** [comment]
currently we don't provide any.  setting up a custom key for this and sharing it between developers doesn't seem like it's worth the effort.  i can start using my own when creating any tags & release tarballs myself.

wdyt @pierrejoye ?  do you have any gpg keys ready to go ?

**Comment by pierrejoye** [comment]
Hi Mike

I don't.  What should we use? one we share or?

**Reply by vapier** [reply]
we could create & share a keyring, but that would just be the set of public keys that we say is OK to sign releases.  you'd still have to generate & use your own gpg key locally.

i guess next time we need to cut a release, i can do the final steps (tag it, create tarball, etc...) and document the flow.  then you can try it out for future ones.

**Comment by pierrejoye** [comment]
what do you think about 

https://www.agwa.name/blog/post/ssh_signatures
